source/features/conflict-marker.tsx

```diff
@@ -0,0 +1,90 @@
+import select from 'select-dom';
+import React from 'dom-chef';
+import * as api from '../libs/api';
+import {getOwnerAndRepo, getRepoURL} from '../libs/utils';
+import features from '../libs/features';
+import {alert} from '../libs/icons';
+
+interface PRConfig {
+	id: string;
+	owner: string;
+	name: string;
+}
+
+function getPrNumber(pr: string): string {
+	return pr.split('_')[1];
+}
+
+function createQueryFragment(pr: PRConfig): string {
+	return `
+		${api.escapeKey(pr.id)}: repository(owner: "${pr.owner}", name: "${pr.name}") {
+			pullRequest(number: ${getPrNumber(pr.id)}) {
+				mergeable
+			}
+		}
+	`;
+}
+
+function buildQuery(
+	prs: PRConfig[]
+): string {
+	return prs.map(createQueryFragment).join('\n');
+}
+
+function getPRConfig(element: HTMLElement): PRConfig {
+	try {
+		const prTitle = select('a[data-hovercard-type="repository"]', element)!.textContent!;
+		const [owner, name] = prTitle.trim().split('/');
+
+		return {
+			id: element.id,
+			owner,
+			name
+		};
+	} catch (error) {
```

Comment:
```suggestion
	} catch {
```

Reply:
I suggested that this function isn’t needed, so I’m marking this as resolved.